### PR TITLE
Update to gold standards

### PR DESCRIPTION
**Pull request by dushaniw**
## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

File: sfdc37/Package.md [hunk 1]
```diff
@@ -1,102 +1,116 @@
...
 
-3. Create a new Ballerina project by executing the following command.
+## Compatibility
+|                    |    Version     |  
+| :-----------------:|:--------------:| 
+| Ballerina Language | 0.970.0-beta15 |
+|Salesforce REST API |    v37.0      |  
+
+## Sample
+The Salesforce connector can be used to create an Account SObject, execute a SOQL query, and create a Lead SObject.
```
Comment (appellj) [inline sfdc37/Package.md]: Delete this sentence: "The Salesforce connector can be used to create an Account SObject, execute a SOQL query, and create a Lead SObject."

File: sfdc37/Package.md [hunk 2]
```diff
@@ -1,102 +1,116 @@
...
+}
+```
+The `getQueryResult` function executes a SOQL query that returns all the results in a single response, or, if it exceeds 
+the maximum record limit, it returns part of the results and an identifier used to retrieve the remaining results. 
+```ballerina
+string sampleQuery = "SELECT name FROM Account";
+response = salesforceEP->getQueryResult(sampleQuery);
+```
+The response from `getQueryResult` is either a JSON object with total size, execution status, resulting records, and 
+url to get next record set (if query execution was successful) or a `SalesforceConnectorError`
```
Comment (appellj) [inline sfdc37/Package.md]: Change "url" to "URL"

File: sfdc37/Package.md (hunk 1 above)
Comment (dushaniw) [reply sfdc37/Package.md]: Fixed

File: sfdc37/Package.md (hunk 2 above)
Comment (dushaniw) [reply sfdc37/Package.md]: fixed

**Pull request merged**